Netro Sprinklers.indigoPlugin/Contents/Server Plugin/plugin.py

```
@@ -540,35 +608,64 @@ def _update_from_netro(self):
+                        # Already logged detailed error in _make_api_call, just skip this device
+                        pass
+                    except requests.exceptions.HTTPError as exc:
+                        # Check if we already logged a detailed error for this
+                        if hasattr(exc, 'response') and exc.response is not None:
+                            try:
+                                error_data = exc.response.json()
+                                if error_data.get("status") == "ERROR":
+                                    # Already logged specific error in _make_api_call
+                                    pass
+                                else:
+                                    self.logger.error("Error getting user data from Netro via API.")
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)